nginx-config/default.conf

```
@@ -7,7 +7,10 @@ server {
     ssl_certificate_key /etc/letsencrypt/live/brokencrystals.com/privkey.pem;
 
     root         /var/www/html;
-
+    # Add Gzip support
+    gzip on;
+    gzip_types      text/plain application/xml application/json;
```

Enable the compression for the static content as well

```suggestion
    gzip_types
        application/xml
        application/json
        font/eot
        font/otf
        font/ttf
        image/svg+xml
        text/css
        text/javascript
        text/plain
        text/xml;
```

---

seems for static files we need something like: 
```
location / {
    gzip_static on;
}

```

---

If I'm not mistaken `gzip_static` is for precompressed files